src/diffs/package-json.js

```diff
@@ -0,0 +1,43 @@
+// Heavily inspired by https://github.com/greenkeeperio/greenkeeper/blob/8be4bfc45dc991ec0c3331ac15b688728f5938a5/lib/diff-package-json.js
+
+const _ = require('windbreaker-service-util/util/objectHelpers')
```

Comment:
It would probably be better to name it something other than `_` since we might add additional helper functions that aren't associated with lodash or underscore and it might cause some confusion.